### PR TITLE
A few qdel fixes.

### PR DESCRIPTION
**Pull request by xxalpha**
Fixed a few landmarks not qdeling
Fixed a case where lattices would not qdel
Fixed igniters not qdeling

Fixes https://github.com/tgstation/-tg-station/issues/11368
Fixes https://github.com/tgstation/-tg-station/issues/11362

Edit
Fixed Cyborg equipment not qdeling
Fixed RPD not qdeling
Fixes https://github.com/tgstation/-tg-station/issues/11372

### CODE REVIEW

File: code/modules/mining/mine_turfs.dm [hunk 1]
```diff
@@ -602,6 +602,17 @@ var/global/list/rockTurfEdgeCache
 	icon_state = "asteroid_dug"
 	return
 
+/turf/simulated/floor/plating/asteroid/singularity_pull(S, current_size)
+	if(current_size == STAGE_THREE)
```
Comment (Jalleo) [inline code/modules/mining/mine_turfs.dm]: This whould get a comment explaining the reason why its overriding the standard floor. Due to the base turf change kor did a while back now.

File: code/modules/mining/mine_turfs.dm [hunk 2]
```diff
@@ -602,6 +602,17 @@ var/global/list/rockTurfEdgeCache
...
+/turf/simulated/floor/plating/asteroid/singularity_pull(S, current_size)
+	if(current_size == STAGE_THREE)
+		if(prob(30))
+			gets_dug()
+	else if(current_size == STAGE_FOUR)
+		if(prob(50))
+			gets_dug()
+	else if(current_size >= STAGE_FIVE)
+		if(prob(90))
+			gets_dug()
```
Comment (Jalleo) [inline code/modules/mining/mine_turfs.dm]: A second prob should be done here to make the turf change to space. Just to make it more scary when the asteroid is dissapearing around you.

Comment (xxalpha) [inline code/modules/mining/mine_turfs.dm]: It's a pull, it pulls the sand. I'll leave transforming the turf to space to singularity eating.

**Comment by Razharas** [comment]
@xxalpha

**Comment by xxalpha** [comment]
Fixed conflicts.

**Pull request merged**